twemproxy/tests/test_twemproxy.py

```diff
@@ -0,0 +1,156 @@
+import os
```

Comment:
I believe this file needs a license header and `twemproxy.py` also needs its header updated.

Reply:
I fixed that